### PR TITLE
fix: Wait for interfaces after microservice restart in the bridge domain E2E tests

### PR DESCRIPTION
**Pull request by rewenset**
Issue: #1489 

Wait a little bit longer for recreation of af-packet interface
**EDIT**: and for TAP interface too

### CODE REVIEW

**Comment by rewenset** [comment]
Oh common! Now it's  `--- FAIL: TestBridgeDomainWithTAPs (66.80s)`:
https://travis-ci.com/ligato/vpp-agent/jobs/241439615#L8422-L8427
with same error as was for `TestBridgeDomainWithAfPackets`.

File: tests/e2e/040_bridge_domain_test.go [hunk 1]
```diff
@@ -22,10 +22,10 @@ import (
...
 
-	"github.com/ligato/vpp-agent/api/models/linux/interfaces"
-	"github.com/ligato/vpp-agent/api/models/linux/namespace"
-	"github.com/ligato/vpp-agent/api/models/vpp/interfaces"
-	"github.com/ligato/vpp-agent/api/models/vpp/l2"
+	linux_interfaces "github.com/ligato/vpp-agent/api/models/linux/interfaces"
+	linux_namespace "github.com/ligato/vpp-agent/api/models/linux/namespace"
+	vpp_interfaces "github.com/ligato/vpp-agent/api/models/vpp/interfaces"
+	vpp_l2 "github.com/ligato/vpp-agent/api/models/vpp/l2"
 	kvs "github.com/ligato/vpp-agent/plugins/kvscheduler/api"
```
Comment (milanlenco) [inline tests/e2e/040_bridge_domain_test.go]: Aren't these aliases redundant?

Comment (rewenset) [reply tests/e2e/040_bridge_domain_test.go]: one of tools in VSCode insist about this

Comment (rewenset) [reply tests/e2e/040_bridge_domain_test.go]:
```
➜ goimports -d 040_bridge_domain_test.go
diff -u 040_bridge_domain_test.go.orig 040_bridge_domain_test.go
--- 040_bridge_domain_test.go.orig      2019-10-04 11:39:07.309005126 +0300
+++ 040_bridge_domain_test.go   2019-10-04 11:39:07.309005126 +0300
@@ -22,10 +22,10 @@
        . "github.com/onsi/gomega"
        "github.com/onsi/gomega/types"

-       "github.com/ligato/vpp-agent/api/models/linux/interfaces"
-       "github.com/ligato/vpp-agent/api/models/linux/namespace"
-       "github.com/ligato/vpp-agent/api/models/vpp/interfaces"
-       "github.com/ligato/vpp-agent/api/models/vpp/l2"
+       linux_interfaces "github.com/ligato/vpp-agent/api/models/linux/interfaces"
+       linux_namespace "github.com/ligato/vpp-agent/api/models/linux/namespace"
+       vpp_interfaces "github.com/ligato/vpp-agent/api/models/vpp/interfaces"
+       vpp_l2 "github.com/ligato/vpp-agent/api/models/vpp/l2"
        kvs "github.com/ligato/vpp-agent/plugins/kvscheduler/api"
 )
```

Comment (ondrej-fabry) [reply tests/e2e/040_bridge_domain_test.go]: I suspect it's because name of directory does not match package name.

**Pull request merged**